Dynatrace/dynatrace-operator
# Remove additional environment requirements from Cloudnative e2e test

# Description

Tests requiring additional installs on the cluster are removed from the default CloudNative test. There are 4 CN tests:
- cloudnative (`make test/e2e/cloudnative`) - the default one, no additional requirements
- cloudnative/istio  (`make test/e2e/cloudnative/istio`) , istio required
- cloudnative/network (`make test/e2e/cloudnative/network`) , cilium required
- cloudnative/proxy (`make test/e2e/cloudnative/proxy`) , cilium required

`testdata/*` paths are relative to absolute `project root directory` now. The Project Root Directory is detected in the runtime during module initialization phase.

## How can this be tested?
Use `make test/e2e/cloudnative` or `make test/e2e/cloudnative/<name>` commands to start a test. Make sure you have  properly configured cluster.

## Checklist
